It can't be built in Manjaro (Arch-based)

I tried to built in Manjaro. I have CMake and a buil-essential equivalent ([base-devel](https://www.garron.me/en/bits/build-essential-arch-linux.html))  

```
En el fichero incluido desde /home/myuser/Games/bestagon-mines/src/application/gameGui.h:16,
                 desde /home/myuser/Games/bestagon-mines/src/application/game.h:15,
                 desde /home/myuser/Games/bestagon-mines/src/application/game.cpp:18:
/home/myuser/Games/bestagon-mines/build/cmake/../../src/middleware/gui.h:29:9: error: ‘#pragma’ no se permite aquí
   29 | #pragma region event_flags
      |         ^~~~~~
/home/myuser/Games/bestagon-mines/build/cmake/../../src/middleware/gui.h:32:9: error: expected ‘}’ before ‘EVT_POPUP’
   32 |         EVT_POPUP,
      |         ^~~~~~~~~
/home/myuser/Games/bestagon-mines/build/cmake/../../src/middleware/gui.h:24:5: nota: to match this ‘{’
   24 |     {
      |     ^
/home/myuser/Games/bestagon-mines/build/cmake/../../src/middleware/gui.h:33:9: error: ‘#pragma’ no se permite aquí
   33 | #pragma endregion
      |         ^~~~~~~~~
/home/myuser/Games/bestagon-mines/build/cmake/../../src/middleware/gui.h:70:28: error: ‘Manager’ en el espacio de nombres ‘gui’ no nombra un tipo
   70 |         Manager(const gui::Manager& other) = delete;
      |                            ^~~~~~~
/home/myuser/Games/bestagon-mines/build/cmake/../../src/middleware/gui.h:71:14: error: ‘Manager’ en el espacio de nombres ‘gui’ no nombra un tipo
   71 |         gui::Manager& operator=(const gui::Manager&);
      |              ^~~~~~~
/home/myuser/Games/bestagon-mines/build/cmake/../../src/middleware/gui.h:73:1: error: expected declaration before ‘}’ token
   73 | }
      | ^
/home/myuser/Games/bestagon-mines/src/application/gameGui.h:53:1: error: expected class-name before ‘{’ token
   53 | {
      | ^
/home/myuser/Games/bestagon-mines/src/application/gameGui.h:57:25: error: ‘EVT__begin_custon’ is not a member of ‘gui’; did you mean ‘EVT__begin_system’?
   57 |         EVT_BACK = gui::EVT__begin_custon
      |                         ^~~~~~~~~~~~~~~~~
      |                         EVT__begin_system
/home/myuser/Games/bestagon-mines/src/application/gameGui.h:61:16: error: ‘gui::Manager’ no se ha declarado
   61 |     using gui::Manager::Manager; // same ctors as gui::Manager
      |                ^~~~~~~
/home/myuser/Games/bestagon-mines/src/application/gameGui.h:79:28: error: ‘EVT__begin_custon’ is not a member of ‘gui’; did you mean ‘EVT__begin_system’?
   79 |     EVT_RESET_CLICK = gui::EVT__begin_custon,
      |                            ^~~~~~~~~~~~~~~~~
      |                            EVT__begin_system
/home/myuser/Games/bestagon-mines/src/application/gameGui.h:87:1: error: expected class-name before ‘{’ token
   87 | {
      | ^
/home/myuser/Games/bestagon-mines/src/application/gameGui.h:89:16: error: ‘gui::Manager’ no se ha declarado
   89 |     using gui::Manager::Manager; // same ctors as gui::Manager
      |                ^~~~~~~
/home/myuser/Games/bestagon-mines/src/application/game.cpp: In constructor ‘Game::Game()’:
/home/myuser/Games/bestagon-mines/src/application/game.cpp:101:159: error: no matching function for call to ‘GameGui::GameGui(Game*)’
  101 | rstClick(true), m_fieldVariant(FV_HEX2), m_relNMines(default_relNMines)
      |                                                                       ^

/home/myuser/Games/bestagon-mines/src/application/gameGui.h:86:7: nota: candidate: ‘constexpr GameGui::GameGui()’
   86 | class GameGui : public gui::Manager
      |       ^~~~~~~
/home/myuser/Games/bestagon-mines/src/application/gameGui.h:86:7: nota:   el candidato espera 0 argumentos, se proporcionaron 1
/home/myuser/Games/bestagon-mines/src/application/game.cpp: In member function ‘virtual bool Game::OnUserCreate()’:
/home/myuser/Games/bestagon-mines/src/application/game.cpp:131:14: error: ‘init’ no es un miembro de ‘GameGui’
  131 |     GameGui::init();
      |              ^~~~
make[2]: *** [CMakeFiles/bestagon-mines.dir/build.make:76: CMakeFiles/bestagon-mines.dir/home/myuser/Games/bestagon-mines/src/application/game.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/bestagon-mines.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
  --=====#   build bestagon-mines - failed   #=====--

```

I searched for the pragma error and I found this bug report:

https://gcc.gnu.org/bugzilla/show_bug.cgi?id=60875

that in turn sent me here:

https://bugs.llvm.org/show_bug.cgi?id=41514#c1 

Could it be that the code was malformed? I don't program in C++ since my first years of college.

--- a/src/middleware/gui.h
+++ b/src/middleware/gui.h
@@ -23,17 +23,16 @@ namespace gui
     enum EVENT
     {
         EVT__begin_system = 0,
+
         EVT_NONE = 0,
 
-        // flags, not events
-#pragma region event_flags
 
-        // indicating that the gui needs full control over PGE
-        EVT_POPUP,
-#pragma endregion
+    // flags, not events
+        EVT_POPUP, // indicating that the gui needs full control over PGE
+
 
         EVT__end_system,
-        EVT__begin_custon = 10000
+        EVT__begin_custon = 10000,
     };
 
     class Manager

